src/main/java/uk/co/ramp/policy/alert/AlertChecker.java

```diff
@@ -47,24 +56,36 @@
                         .nextStatus(REQUESTED_TEST)
                         .build());
 
-    var startTime = currentTime - tracingPolicy.recentContactsLookBackTime() + 1;
+    var tracingPolicyItem = findPolicyItem(currentReporterVirusStatus, currentReporterAlertStatus);
+
+    if (tracingPolicyItem.isEmpty()) {
+      return requestTestForReporterEvent;
+    }
+    var startTime = currentTime - tracingPolicyItem.get().recentContactsLookBackTime() + 1;
 
     var alertsFromRecentContacts =
         alertContactTracer.traceRecentContacts(startTime, currentTime, personId).stream()
-            .filter(a -> currentReporterAlertStatus == tracingPolicyReporterAlertStatus)
-            .filter(a -> nextVirusStatus == tracingPolicyReporterVirusStatus)
+            .filter(a -> shouldPerformTracingForThisLink(tracingPolicyItem.get()))
```

Comment:
Similar question to the above.  I'm clearly missing something here!

Reply:
I can't see my "above" comment.  The question is what is this filter doing?  I find it strange that `a` seems to be unused.

Reply:
Hi @johnnonweiler, it's carrying out filtering which is based on external variables rather than the recent contact itself. I will update this so that it is more clear in a follow up PR. Thanks